aospy/calc.py

```diff
@@ -374,9 +375,11 @@ def _add_grid_attributes(self, ds, n=0):
                 except ValueError:
                     ds = ds
                 ds = ds.set_coords(name_int)
-                if not ds[name_int].equals(model_attr):
-                    warnings.warn("Model coordinates for '{}' "
-                                  "do not match those in Run".format(name_int))
+                if not np.array_equal(ds[name_int], model_attr):
```

Comment:
What was the reason for changing this from the DataArray equals method to the numpy array one?  The potential for slightly different coordinate indexes, but the same array values?  (I'm assuming you had a good reason for changing this; I'd just like to know for future reference).